### PR TITLE
Count number of panicking tasks

### PR DESCRIPTION
**Pull request by aidanhs**
Partial solution to #13 (a possible more complete solution would be for the thread pool to have an alternative 'task' api which would return a task status struct containing atomic bools for the status of the corresponding task allowing you to look up specific task statuses, but that's much more than I need right now).

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -104,6 +111,7 @@ pub struct ThreadPool {
     job_receiver: Arc<Mutex<Receiver<Thunk<'static>>>>,
     active_count: Arc<AtomicUsize>,
     max_count: Arc<AtomicUsize>,
+    panicked_count: Arc<AtomicUsize>,
```
Comment (frewsxcv) [inline src/lib.rs]: Any opinions about just calling this `panic_count`?

Comment (frewsxcv) [inline src/lib.rs]: Either way, just throwing out another option

Comment (aidanhs) [inline src/lib.rs]: Shorter name, can't see any downsides...will change.

**Comment by frewsxcv** [comment]
Looks great, thanks for your contribution!

**Comment by frewsxcv** [comment]
threadpool version 1.2.0 has just been released which includes your changes. Thanks again!

**Comment by aidanhs** [comment]
Thanks for the quick release!

**Pull request merged**